src/plugins/appeals/unbanThread.ts

```diff
@@ -0,0 +1,35 @@
+import {useEvent} from "../../hooks";
+import {Interaction, Message} from "discord.js";
+
+useEvent("interactionCreate", async (interaction: Interaction) => {
+    if (!interaction.isButton()) {
+        return;
+    }
+    const {customId} = interaction;
+    const [interactionType, action, userTag, userId] = customId.split("-");
+    const {message} = interaction;
+    if (interactionType !== "appeal") {
+        return;
+    }
+    if (action !== "thread") {
+        return;
+    }
+    try {
+        await interaction.message.startThread ({
+            name: `Appeal Thread - ${userTag}`
+        })
+    } catch {
+        await interaction.reply({
+            content: `A thread was already created for <@${userId}>.`,
```

Comment:
It is better practice to use the `userMention` helper function throughout in case the way Discord formats mentions changes

Reply:
Done.